docs/Tutorial.md

```diff
@@ -158,15 +159,10 @@ There isn't a lot of video tutorials on zmod as its a very specific software to
 
 He has a lot more videos on his channel and its worth a check out if you are looking to get into vehicle development for GTA. 
 
-Now that the basics of zmod has been covered, to create Non ELS vehicles it gets a tad more complicated and is where zmod comes in very handy but can crash very easily. I have a vide on how to create non ELS vehicles that you can watch here: 
+Now that the basics of zmod has been covered, to create Non ELS vehicles it gets a tad more complicated and is where zmod comes in very handy. I have a video on how to create non ELS vehicles that you can watch here: 
```

Comment:
Remember that everything should be from the perspective of an imaginary third party.

> I have a video

Feel free to just shamelessly plug  yourself.
``JackTheDev's Video``